onward/app/weather/controllers/LocationsController.scala

```diff
@@ -34,44 +34,27 @@ class LocationsController(weatherApi: WeatherApi, val controllerComponents: Cont
       def getEncodedHeader(key: String) =
         request.headers.get(key).map(java.net.URLDecoder.decode(_, "latin1"))
 
+      val maybeCountry = getEncodedHeader(CountryHeader).filter(_.nonEmpty)
       val maybeCity = getEncodedHeader(CityHeader).filter(_.nonEmpty)
       val maybeRegion = getEncodedHeader(RegionHeader).filter(_.nonEmpty)
```

Comment:
Why reorder these?

Reply:
Country / city are needed, and nearly always available. Region is way more Maybe like. It's just how my brain was thinking of it, and the order we use in the methods `(country: String, city: String, maybeRegion: Option[String])`.